crates/cxx-qt/src/lib.rs

```diff
@@ -31,3 +35,28 @@ pub trait CxxQtType {
     /// The property changed signal must be emitted manually.
     unsafe fn rust_mut(self: core::pin::Pin<&mut Self>) -> core::pin::Pin<&mut Self::Rust>;
 }
+
+/// Indicates that the object implements threading and has a method which returns a [CxxQtThread]
```

Comment:
I think this documentation could be a bit more descriptive :)

```suggestion
/// Indicates that the object implements threading and has a method which returns a [CxxQtThread].
///
/// This trait is implemented by CxxQt automatically.
/// To enable this for a `qobject::T`, add `impl cxx_qt::Threading for qobject::T {}` to your [`#[cxx_qt::bridge]`](cxx_qt::bridge).
```
Please check that linking to cxx_qt::bridge works that way. :sweat_smile:

Reply:
```
warning: unresolved link to `cxx_qt::bridge`
  --> crates/cxx-qt/src/lib.rs:42:117
   |
42 | /// To enable this for a `qobject::T`, add `impl cxx_qt::Threading for qobject::T {}` to your [`#[cxx_qt::bridge]`](cxx_qt::bridge).
   |                                                                                                                     ^^^^^^^^^^^^^^ no item named `cxx_qt` in scope
   |
   = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default
```
not quite, i'll see if i can find the right thing...

Reply:
```
[`#[cxx_qt::bridge]`](bridge)
```
works as `bridge` is in scope